doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -1094,6 +1094,7 @@ Documentation
   * The :ref:`dev-model` section with the :ref:`documentation` pages, previously listed separately.
   * The :ref:`glossary` page by moving it back to the main hierarchy level.
   * The structure of the :ref:`ug_wifi` page by moving :ref:`ug_nrf70_developing_powersave`, :ref:`ug_nrf70_developing_regulatory_support`, :ref:`ug_nrf70_developing_scan_operation`, :ref:`ug_nrf70_developing_debugging`, :ref:`ug_nrf70_developing_raw_ieee_80211_packet_transmission`, and :ref:`nRF70_soft_ap_mode` pages, which were previously listed under :ref:`ug_nrf70_developing`.
+  * The :ref:`ug_radio_fem_sw_support_mpsl` section in the :ref:`ug_radio_fem` guide, clarifying that the Simple GPIO implementation is intended for multiple front-end modules and not just one specific device.
```

Comment:
This entry might stand out better just before the Protocols section, under heading Working with RF front-end modules. Something like: "Added a clarification that the .... in the :ref:`ug_radio_fem_sw_support_mpsl` section.

Reply:
updated